Sources/Sentry/SentryAppStartTracker.m

```diff
@@ -35,7 +35,9 @@
 @property (nonatomic, assign) BOOL wasInBackground;
 @property (nonatomic, strong) NSDate *didFinishLaunchingTimestamp;
 @property (nonatomic, assign) BOOL enablePreWarmedAppStartTracing;
-
+#    if TEST
+@property (nonatomic) BOOL isRunning;
+#    endif
```

Comment:
`m`: This class is internal; I would make this a public readonly property for SentryAppStartTracker. Then we don't need to use Dynamic for testing. You can keep the `Test` compiler directive, but I think removing it would also be fine.